pages/doc.js

```diff
@@ -174,7 +213,6 @@ export default class Documentation extends Component {
       containerId: 'bodybag'
     })
   }
-
```

Comment:
why are we removing this?

Reply:
we have removed the spaces for greater validity in CodeClimate